### PR TITLE
*DO NOT MERGE* fikid Naif Keywords

### PR DESCRIPTION
**Pull request by acpaquette**
Adds any naif keywords that are associated with the fikid of an instrument.

This fix was necessary for messenger to be able to spiceinit with ISIS.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -20,14 +20,20 @@ find_package(Eigen3 3.3    REQUIRED NO_MODULE)
...
+
 # Library setup
 add_library(ale SHARED
             ${CMAKE_CURRENT_SOURCE_DIR}/src/ale.cpp
             ${CMAKE_CURRENT_SOURCE_DIR}/src/Rotation.cpp)
 set(ALE_BUILD_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/include/")
 set(ALE_HEADERS "${ALE_BUILD_INCLUDE_DIR}/ale.h"
                 "${ALE_BUILD_INCLUDE_DIR}/Rotation.h")
-set(ALE_INSTALL_INCLUDE_DIR "include/ale")
+set(ALE_INSTALL_INCLUDE_DIR "include/")
```
Comment (jessemapel) [inline CMakeLists.txt]: Why are you changing the directory we are installing into? Some of our headers have very common names that may clash with other packages if they are installed directly into the include directory.

File: tests/pytests/test_mex_drivers.py [hunk 2]
```diff
@@ -509,6 +572,7 @@ def test_mex_load(test_kernels, formatter, usgscsm_compare_dict, label):
 
         usgscsm_isd = ale.load(label_file, props={'kernels': test_kernels}, formatter=formatter, verbose='true')
         assert compare_dicts(usgscsm_isd, usgscsm_compare_dict['h5270_0000_ir2'][formatter]) == []
+        assert False
```
Comment (jessemapel) [inline tests/pytests/test_mex_drivers.py]: This looks like old debug stuff.

File: CMakeLists.txt (hunk 1 above)
Comment (acpaquette) [reply CMakeLists.txt]: :+1: This can be reverted

**Comment by acpaquette** [comment]
Closing this PR so someone else could potentially pick up the work from my fork. This PR fails intermittently due to how python handles putting the found naif keywords into a dictionary.

**Pull request closed without merging**